### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NimaVaziri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,11 @@
...
-<p align="center"><b>
-🚧 Under active development. EIGENSDK-GO is rapidly being upgraded, features are being added, interfaces will have breaking changes 🚧
-</b><p>
 
-**Do not use it in Production, testnet only.**
+
 
 ## EigenSDK
-This SDK provides a set of primitive Go modules for developing AVSs used in EigenLayer
+This SDK provides a set of primitive Go modules for developing AVSs used in EigenLayer.
```
Comment (samlaf) [inline README.md]: « in Eigenlayer » sounds a bit weird to me. Is there another formulation we could use instead?

**Review by samlaf — DISMISSED** [review]
LGTM. I’d change one sentence highlighted below but otherwise good to go.

File: README.md (hunk 1 above)
Comment (NimaVaziri) [inline README.md]:
```suggestion
This SDK provides a set of primitive Go modules for developing AVSs on EigenLayer.
```

Comment (NimaVaziri) [reply README.md]: Agreed, changed to "on" vs "used in"

**Review by samlaf — APPROVED** [review]

**Pull request merged**